rcl/test/rcl/test_timer.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
Is this a new file and not a moved file? If so should this be 2017?

Reply:
it's a new file and indeed should be updated :+1: